src/main/java/clickup/FoldersRequests.java

```diff
@@ -0,0 +1,39 @@
+package clickup;
+
+import core.api.ApiManager;
+import core.api.ApiMethod;
+import core.api.ApiRequest;
+import core.api.ApiRequestBuilder;
+import core.api.ApiResponse;
+
+public final class FoldersRequests {
+    private FoldersRequests() {
+    }
+
+    public static ApiResponse getFolder(final String folderId) {
```

Comment:
Add javadoc

Reply:
I realized, I was not using the class so I deleted it. Thanks.